app/policies/article_policy.rb

```diff
@@ -1,4 +1,23 @@
 class ArticlePolicy < ApplicationPolicy
+  class Scope
```

Comment:
I think it's more convenient to place Scope at the bottom of policy

Reply:
fixed